diff --git a/.next/cache/.previewinfo b/.next/cache/.previewinfo
deleted file mode 100644
index eb6aa0f..0000000
--- a/.next/cache/.previewinfo
+++ /dev/null
@@ -1 +0,0 @@
-{"previewModeId":"798892d50e4c8e4ddb5204488f9e2ed6","previewModeSigningKey":"e3b5186822ced9ba07c69803827660c609fb842f8d7457f9737967b08f3359af","previewModeEncryptionKey":"3cab3f51e5c657f3e58ef57ffc278a13aa707a1cc3715a002b1fc98b71ded6e0","expireAt":1778552725568}
\ No newline at end of file
diff --git a/.next/cache/.rscinfo b/.next/cache/.rscinfo
deleted file mode 100644
index 9acfa4c..0000000
--- a/.next/cache/.rscinfo
+++ /dev/null
@@ -1 +0,0 @@
-{"encryption.key":"yry/16uAk48YpZ5SboyfNDHDHSeJky13peWw1Jw+JNk=","encryption.expire_at":1778552725523}
\ No newline at end of file
